templates/default/cadvisor.service.erb

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description = cadvisor
+StartLimitIntervalSec=30
+StartLimitBurst=20
+
+[Service]
+ExecStart = <%= @cadvisor_bin %> \
+--storage_driver_host=localhost:8086 \
+--storage_driver_password=root \
```

Comment:
Is it a security risk by exposing this password here? Are users able to connect to it and what information can they get?